monitors/index.js

```diff
@@ -56,12 +56,24 @@ async function run(
       medianizerPriceFeedConfig
     });
 
-    // Setup web3 accounts an contract instance
-    const accounts = await web3.eth.getAccounts();
+    // 0. Setup EMP and token instances to monitor.
     const emp = await ExpiringMultiParty.at(address);
+    const collateralTokenAddress = await emp.collateralCurrency();
+    const collateralToken = await ExpandedERC20.at(collateralTokenAddress);
+    const syntheticTokenAddress = await emp.tokenCurrency();
+    const syntheticToken = await ExpandedERC20.at(syntheticTokenAddress);
+
+    // Generate EMP properties to inform monitor modules of important info like token symbols and price identifier.
+    const empProps = {
+      collateralCurrencySymbol: await collateralToken.symbol(),
+      syntheticCurrencySymbol: await syntheticToken.symbol(),
+      priceIdentifier: hexToUtf8(await emp.priceIdentifier()),
+      networkId: await web3.eth.net.getId()
+    };
 
-    // Setup price feed.
-    // TODO: consider making getTime async and using contract time.
+    console.log("EMPPROPOS", empProps);
```

Comment:
nit: did you mean to remove this?

Reply:
ye, that was a mistake. I have removed it.